setup.py

```diff
@@ -37,7 +37,7 @@
     pass
 
 # Other requirements
-install_requires += ["scipy"]
+install_requires += ["scipy", "jaxtyping>=0.2.9"]
```

Comment:
I think we can do this. It's a bit tricky because the signatures all use Float from jaxtyping but I believe this is all optional stuff. I have moved this to extras_require in setup.py.

Reply:
Never mind... now that I've played around with it, it seems like it would be a pain to make this an optional dependency!